pyquil/quilbase.py

```diff
@@ -1110,7 +1114,7 @@ def __repr__(self) -> str:
         return "<PRAGMA {}>".format(self.command)
 
 
-class Declare(AbstractInstruction):
+class Declare(quil_rs.Declaration, AbstractInstruction):
```

Comment:
Unfortunate that this needed so much wrapping from the python side

Reply:
Yeah 😞